global.json

```diff
@@ -1,18 +1,18 @@
 {
   "sdk": {
-    "version": "10.0.100-preview.3.25201.16",
+    "version": "10.0.100-preview.6.25272.112",
```

Comment:
Is there a reason we need to update this to an unreleased preview, or could we wait until the preview is released? MacOS has limitations on test signed sdks and the C# Dev Kit compatibility workaround is broken, so the VSCode experience on Macs is not great with this version.

Reply:
do we have an issue for the VSCode workaround being broken? I assume you're talking about 
<img width="1033" alt="Screenshot 2025-05-23 at 22 33 21" src="https://github.com/user-attachments/assets/496eb276-5a71-4c3a-a53c-6262aca6429f" />

Reply:
Yes, that's the workaround I was talking about. I filed https://github.com/microsoft/vscode-dotnettools/issues/2082 on dev kit.